src/pygama/dsp/_processors/pole_zero.py

```diff
@@ -49,7 +49,10 @@ def pole_zero(w_in, t_tau, w_out):
 def double_pole_zero(w_in, t_tau1, t_tau2, frac, w_out):
     """
     Apply a double pole-zero cancellation using the provided time
-    constant to the waveform.
+    constant to the waveform. This algorithm is a IIR filter to deconvolve the function
+    s(x) = A[frac*exp(-t/t_tau2) + (1-frac)*exp(-t/t_tau1)] into a single step function
+    of amplitude A. See the June 1st analysis and simulations call for more details.
+    https://indico.legend-exp.org/event/840/
 
```

Comment:
Hi @SamuelBorden, can I ask you to improve the documentation here? I would a) use the [correct syntax for displaying LaTeX strings](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#math) and b) describe the algorithm without referring to external and non publicly accessible documentation. You can find further instructions in our [developer guide](https://legend-exp.github.io/pygama/refactor/contributing.html#documentation).